### ISSUE TITLE
Allow to disable spaces between documentation parts dynamically ?

### ISSUE THREAD

**Issue by ncoden** [20Tauri/DoxyDoxygen#8]
Configuration to choose when to have spaces or not between the parts of the documentation.
For exemple, enable/disable split around the @param list when there is only 1 parameter...

Or... (To avoid 15,000 doc styles, and if there is no other ideas of personalisation), always disable the split around the @param list when there is only 1 parameter.
Or... Disable this split by default ?

**Comment by 20Tauri** [comment]
I'm not sur to understand, it's and existing setting:

``` json
    "prefered_tabs_sizes": [ 1 ],
    "min_spaces_between_columns": [ 1 ],
```

If not please give me some examples

**Comment by ncoden** [comment]
Example
Disable the empty lines around a single @param to have

```
/**
** @brief      Test if the character is alphabetic or numeric
** @param      c     Character to test
** @return     Return TRUE if the character is alphabetic or numeric, FALSE
**             otherwise.
*/
```

instead of

```
/**
** @brief      Test if the character is alphabetic or numeric
**
** @param      c     Character to test
**
** @return     Return TRUE if the character is alphabetic or numeric, FALSE
**             otherwise.
*/
```

**Comment by 20Tauri** [comment]
You probably use "Doxygen".
Simply set "block_layout" by removing "space".
It's very flexible...
But suggestion to improve documentation are welcome

**Comment by ncoden** [comment]
I don't want to disable spaces for all situation, but only when there is only 1 parameter
And i don't understand "You probably use Doxygen.". I use Doxygen to generate my doc files, but it's your plugin wich, in SublimeText, generate the doc text, with _always_ a space around one single @param

**Comment by 20Tauri** [comment]
I haven't understand the need. Sorry.

DoxyDoxygen supports a lot of DocStyles (documentation tool). In the default configuration, spaces between @param and the others tags may be by default removed... That's why I suppose you use "Doxygen"

You may currently chosse to
- have spaces (or not) before the @param group
- have spaces (or not) after the @param group

But configuration doesn't care the number of @param.
Is there someone else interested by this feature ?
Is it a "standard" in your school ?

**Comment by 20Tauri** [comment]
This will be fix in 0.27.0.
I will update the documentation and you will be able to create our own custom rule

**Comment by 20Tauri** [comment]
Released in standard edition
